packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -121,7 +122,9 @@ void Multiply(
 #ifdef HAVE_TPETRA_MMM_TIMINGS
   std::string prefix_mmm = std::string("TpetraExt ") + label + std::string(": ");
   using Teuchos::TimeMonitor;
-  TimeMonitor MM(*TimeMonitor::getNewTimer(prefix_mmm + std::string("MMM All Setup")));
```

Comment:
I reverted all the semantic changes to Multiply(), but the timer-related changes should stay. "MMM All Setup" was timing the entire function body, but it should really just be timing everything up to the multiply.